buildsystem/spack/summit/spack.yaml

```diff
@@ -1,11 +1,12 @@
 spack:
   specs:
   - exago@develop%gcc@10.2.0+cuda+mpi+hiop+raja+ipopt+python cuda_arch=70 build_type=Release
-    ^hiop@0.7.2+cuda+mpi+raja+kron+sparse+ginkgo+cusolver_lu cuda_arch=70
+    ^hiop@develop+cuda+mpi+raja+kron+sparse+ginkgo+cusolver_lu cuda_arch=70
+    ^cub@1.16.0
```

Comment:
Is this cub dependency summit specific, or should this be expressed in our package.py somehow?

Reply:
Good catch. It's no longer needed on Summit either. There was a compatibility issue between cub and the old cuda modules on Summit, so I had experimented with different cub versions. Now, I'm letting Spack build cuda, so I no longer need to enforce a specific cub version. If you don't mind, I'll remove in the next PR where I'll install the modules in ENG142 directories.

Reply:
Great